### ISSUE TITLE
Another small suggestion: potential misnomer 

### ISSUE THREAD

**Issue by Mr-2c** [ExtremeFLOW/neko#1398]
Naming is a bit confusing (especially for new users). Perhaps this 'u%fluid_user_if' can be renamed throughout Neko since it applies to all BCs with a Dirichlet type and not only inflow conditions. 
https://github.com/ExtremeFLOW/neko/blob/66eea02d3be5558fc5172132a45251a17bec8f0a/examples/cyl_boundary_layer/cyl_bl.f90#L27C5-L27C20

**Comment by timofeymukha** [comment]
Yeah, we will need to go through these things with the bcs, once the new interface gets merged. Hopefully some time in the fall :-).

**Comment by MartinKarp** [comment]
I think with the addition of dirichlet_update, this one can be removed? Would remove the potential for setting bcs that cannot be changed in time.

**Comment by MartinKarp** [comment]
I would I guess like to remove a lot of the user functions and focus on a few that really work. Right now the old ones do not really help with "quick and dirty" thing either in my opinion

**Comment by timofeymukha** [comment]
I agree with you, "quick and dirty" to me is being closer to "god mode", and the field_dirichlet ones are definitely more in that direction.

**Comment by MartinKarp** [comment]
Yeah, I think limiting ourselves to a few very powerful functions in the user file would be better than tons with quite niche functionality and that do not have accesss to for examples the case or similar
